pyclient/setup.py

```diff
@@ -1,20 +1,34 @@
 #
 #  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
 #
+import pathlib
+from setuptools import find_packages, setup
 
-from setuptools import setup
+# The directory containing this file
+HERE = pathlib.Path(__file__).parent
+
+# The text of the README file
+README = (HERE / "README.md").read_text()
 
 setup(
     name='pydeephaven',
-    version='0.0.1',
-    packages=['pydeephaven'],
+    version='0.4.0',
+    description='The Deephaven Python Client',
+    long_description=README,
+    long_description_content_type="text/markdown",
+    packages=find_packages(exclude=("tests",)),
     url='https://deephaven.io/',
     license='Deephaven Community License Agreement Version 1.0',
     author='Deephaven Data Labs',
-    author_email='jianfengmao@deephaven.io',
-    description='The Deephaven Python Client',
+    author_email='python@deephaven.io',
```

Comment:
Does this email address exist? I know we are/have created `developers@deephaven.io`.

Reply:
it is used in the other two DH projects on PYPI